java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -369,4 +373,44 @@ CompletableFuture<Long> linsert(
      * }</pre>
      */
     CompletableFuture<Long> lpushx(String key, String[] elements);
+
+    /**
+     * Pops one or more elements from the first non-empty list from the list of provided <code>key
+     * </code>
+     *
+     * @since Redis 7.0 and above.
+     * @apiNote When in cluster mode, all <code>keys</code> must map to the same hash slot.
+     * @see <a href="https://valkey.io/commands/lmpop/">valkey.io</a> for details.
+     * @param keys The list of provided <code>key</code> names.
+     * @param direction The direction based on which elements are popped from - see {@link
+     *     PopDirection}.
+     * @param count The count of elements to be popped and returned.
+     * @return A <code>Map</code> of <code>key</code> names arrays of popped elements.
```

Comment:
Something is missing, please reword
```suggestion
     * @return A <code>Map</code> of <code>key</code> names mapped arrays of popped elements.
```